Makefile

```diff
@@ -2,7 +2,7 @@ UNITS = "@unit.abijson or @unit.abijson.byname or @unit.algod or @unit.algod.led
 unit:
 	behave --tags=$(UNITS) tests -f progress2
 
-INTEGRATIONS = "@abi or @algod or @applications or @applications.verified or @assets or @auction or @c2c or @compile or @dryrun or @dryrun.testing or @indexer or @indexer.231 or @indexer.applications or @kmd or @rekey or @send.keyregtxn or @send or @compile.sourcemap"
+INTEGRATIONS = "@abi or @algod or @applications or @applications.verified or @assets or @auction or @c2c or @compile or @dryrun or @dryrun.testing or @indexer or @indexer.231 or @indexer.applications or @kmd or @send.keyregtxn or @send or @compile.sourcemap"
```

Comment:
The existing v1 rekeying tests alters the default wallet key - I decided to just remove it here, but a more complete solution would be to re-write them so that they alter a transient private key instead.

Reply:
Do you know if this problem is universal across SDK's?

Reply:
> Do you know if this problem is universal across SDK's?

I haven't looked into all of them, but I think it would be. I also think the order of execution matters - it poses the least amount of problems if `@rekey` is executed last.

I did push a workaround hack for the v1 functions in my latest commit - I created a transient dev account with some funds to send transactions before a potential rekey.